ttt/minimax.rb

```diff
@@ -0,0 +1,75 @@
+require_relative 'ai'
+require_relative 'symbols'
+
+class Minimax
+  @board
+  @ai_symbol
+  @current_symbol
+  @human_symbol
+  @winner
+  @symbols
+
+  def initialize(board, symbols)
+    @board = board
+    @symbols = symbols
+    @ai_symbol = symbols.get_ai
+    @human_symbol = symbols.get_human
+  end
+
+  def get_best_move
+    depth = 1
+    best_move = 0
+    best_score = -100
```

Comment:
You can create constant like this:

```rb
class Minimax
  BEST_SCORE = -100
end
```

You can reference it within the same class with `BEST_SCORE`, or outside of the class with `Minimax::BEST_SCORE`.

Reply:
the `best_score` here keeps changing its not a constant. will pull out other constants though 👍